### PR TITLE
User Login/Signup -Working

### PR DESCRIPTION
**Pull request by DexVitalitY**

### CODE REVIEW

File: app/assets/javascripts/angular/controllers/users/landing_page.js.erb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+app.controller('LandingPageCtrl', ['$scope', '$http', '$location', function($scope, $http, $location){
```
Comment (f3r) [inline app/assets/javascripts/angular/controllers/users/landing_page.js.erb]: if it's empty, perhaps better to add dependencies later -- when/if you need them

``` js
app.controller('LandingPageCtrl', [function(){
}]);
```

File: app/assets/javascripts/angular/controllers/users/landing_page_signin.js.erb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+app.controller('LoginCtrl', ['$scope', '$http', '$location', function($scope, $http, $location){
+	
+  $scope.user = {};
+  $scope.form = {};
+
+  $scope.form.login = function(){
+    console.log($scope.user);
```
Comment (f3r) [inline app/assets/javascripts/angular/controllers/users/landing_page_signin.js.erb]: once it works, you should remove all `console.log`s that are not necessary

File: app/assets/stylesheets/users.scss [hunk 3]
```diff
@@ -0,0 +1,11 @@
+.fer {
```
Comment (f3r) [inline app/assets/stylesheets/users.scss]: pls delete

File: app/assets/templates/users/landing.html [hunk 4]
```diff
@@ -0,0 +1,60 @@
+<!-- <head>
+  <link href="landing.css" rel="stylesheet" type="text/css">
+</head> -->
+
+<div class="container">
+  <div class="row">
+    <div class="description col-xs-6 col-xs-offset-3 col-sm-10 col-sm-offset-1">
+      
+      <p class="fer">This Is Where The Log-in/SignUp Information Needs To Be</p>
```
Comment (f3r) [inline app/assets/templates/users/landing.html]: remove class

File: app/assets/stylesheets/home.scss [hunk 5]
```diff
@@ -1,3 +1,4 @@
 // Place all the styles related to the home controller here.
```
Comment (h8rry) [inline app/assets/stylesheets/home.scss:1]: You can remove this file if you don't use it

File: app/assets/stylesheets/users.scss [hunk 6]
```diff
@@ -0,0 +1,9 @@
+
```
Comment (h8rry) [inline app/assets/stylesheets/users.scss]: No line breaks if not needed

File: app/assets/templates/users/landing.html [hunk 7]
```diff
@@ -0,0 +1,59 @@
+
```
Comment (h8rry) [inline app/assets/templates/users/landing.html:1]: No line breaks if not needed. Please check through the files in this PR.

File: app/assets/javascripts/angular/controllers/users/profile.js.erb [hunk 8]
```diff
@@ -0,0 +1,13 @@
+app.controller('UserProfileCtrl', ['$scope', '$http', '$location', function($scope, $http, $location){
+  
+  var url = "<%= ENV['URL'] %>" || "http://localhost:3000/";
+
+  $http.get(url + 'users/' + current_user.id + '/posts', data).success(function(response){
```
Comment (h8rry) [inline app/assets/javascripts/angular/controllers/users/profile.js.erb]: Please don't put this here if it doesn't work :)

**Comment by h8rry** [comment]
Looks good

**Pull request merged**